app/controllers/doctors_controller.rb

```diff
@@ -30,28 +31,34 @@ def edit
 end
 
 def update
+  @doctor = Doctor.find(params[:id])
+  @doctor.update(doctor_params)
+  redirect_to "doctors/index"
+end
+
+
+def new_visit
+  @doctor = current_user.patients.find(params[:id])
```

Comment:
Where is `current_user` defined? I don't see it in the files- does this already work or does this require creating a new method for `current_user`?

Reply:
Resolved